pkg/connector/destination.go

```diff
@@ -18,11 +18,12 @@ import (
 	"context"
 	"sync"
 
+	"github.com/conduitio/conduit-commons/opencdc"
+	"github.com/conduitio/conduit-connector-protocol/pconnector"
 	"github.com/conduitio/conduit/pkg/foundation/cerrors"
 	"github.com/conduitio/conduit/pkg/foundation/log"
 	"github.com/conduitio/conduit/pkg/plugin"
 	connectorPlugin "github.com/conduitio/conduit/pkg/plugin/connector"
-	"github.com/conduitio/conduit/pkg/record"
 )
 
 type Destination struct {
```

Comment:
The code in `connector.Destination` and `connector.Source` needs a bit more attention than the rest. We removed an abstraction layer in `pkg/plugin/connector/...`, this code now does just a bit more, as it communicates more directly with the plugin.